src/components/CardList/CardList.tsx

```diff
@@ -1,13 +1,14 @@
+import React, { ReactElement } from "react";
```

Comment:
Don't import whole module and types separately.  only import what you need like this - import { ReactElement, useState } from "react";